packages/upswyng-web/src/components/Schedule.tsx

```diff
@@ -138,12 +107,12 @@ const Schedule = ({ schedule }: ScheduleProps) => {
     // group together schedule items that have the same comments
     const commentMap = resourceSchedule.getItems().reduce((result, item) => {
       const comment = item.comment || "_no_comment_";
-      result[comment as keyof typeof dayToCode] = [
-        ...(result[comment as keyof typeof dayToCode] || []),
+      result[comment as keyof typeof days] = [
```

Comment:
i think you can prob get rid of all this type info if you're just gonna use an array for `days`.. `keyof typeof days` reduces to `number`

Reply:
TS may fight you tho

Reply:
Outdated